### PR TITLE
[feat-5240]: 'instellingen' is included in url of nested pages

### PR DESCRIPTION
**Pull request by SireeKoolenWijkstra**
**Context**
In the backoffice in Instellingen you can click on several links. When clicked the url was directly appended to the main url and not appended to main url/instellingen/. We want the url to reflect the nesting. Instellingen and not Settings was chosen, because the website is in Dutch.

**Changes**
- In App/index at the highest level, there were two directs to the Settingsmodule, one with the wildcard instellingen/* and one as a catch all *. The catch all directs now to the not-found page.
- In order to append to 'instellingen/', the strings in route.js in the setting directory are now all without /.
- All navigate to's now navigate to base_url + / + whatever the rest of the url is 

Ticket: [SIG-5240](https://gemeente-amsterdam.atlassian.net/browse/SIG-5240)

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

**Review by OscarBakker — CHANGES_REQUESTED** [review]
Looks good. Just a few things

File: src/signals/settings/roles/containers/RolesListContainer/index.js [hunk 1]
```diff
@@ -27,34 +27,40 @@ const HeaderButton = styled(Button)`
...
-          <RolesList
-            list={list}
-            linksEnabled={Boolean(userCan('change_group'))}
-          />
-        )}
-      </Column>
-    </Row>
-  </Fragment>
-)
+}) => {
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply src/signals/settings/roles/containers/RolesListContainer/index.js:30]: Removed Fragment

File: src/signals/settings/routes.ts [hunk 2]
```diff
@@ -1,21 +1,25 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2019 - 2023 Gemeente Amsterdam
 
+/*
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply src/signals/settings/routes.ts:4]: Removed it

File: src/components/pages/BasePage/BasePage.tsx [hunk 3]
```diff
@@ -37,27 +35,29 @@ const BasePage = ({
...
-            <StyledHeading>{pageTitle}</StyledHeading>
-          </header>
-        )}
-
-        {children}
-      </article>
-    </ContentWrapper>
-  </Row>
-)
+}: BasePageProps) => {
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply src/components/pages/BasePage/BasePage.tsx:38]: I don't understand this. This is what is originally was. I only removed the Default_Message

**Review by OscarBakker — APPROVED** [review]

**Pull request merged**